documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -24,10 +30,19 @@ COPY ./_my-plugins_/ /opt/kafka/plugins/
 USER {DockerImageUser}
 ----
 
-. Build the container image and upload it to the appropriate container image repository.
+. Build the container image.
+
+. Upload your custom image to your container registry.
 
-. Set the `KafkaConnect.spec.image` property of the KafkaConnect custom resource or the `STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE` variable to point to the new container image.
+. Perform one of the following tasks:
```

Comment:
Why are we offering two options? Do they achieve the same thing? If so I say we should be more prescriptive.

Reply:
I'd need some input from @ppatierno on this. ;)

Reply:
Well ... the Cluster Operator (CO) YAML file has DEFAULT for every image that we use for the different containers (Kafka, Zookeeper, Kafka Connect and so on). It means that the CO will use that DEFAULT image if the user doesn't set anything in the `image` field for the specific resource (i.e. if `KafkaConnect.spec.image` is not set, the CO will create a Kafka Connect instance using the `STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE`). As opposite, the `image` property, if set, override the DEFAULT one.